src/main/java/SignUpPage.java

```diff
@@ -16,42 +19,56 @@ public class SignUpPage extends BasePage {
     public By needHelpLink = By.xpath("//*[@class='a-expander-prompt']/ancestor::a");
 
     protected void openPage() {
-        super.openPage(url);
+        log.info("Open the 'SignUp' page");
+        super.openPage(URL);
     }
 
     public void fillUserName(String userName) {
+        log.info("Fill 'userName' field");
```

Comment:
those are fine but for more detailed logs you can use log.info("Fill 'userName' field with "+userName);